File: backend/src/main/java/com/printit/backend/features/admin/orders/AdminOrderResponse.java
package com.printit.backend.features.admin.orders;

public class AdminOrderResponse {

    private Long id;
    private String fileName;
    private String studentName;
    private String dateSubmitted;
    private Long copies;
    private String status;

    public AdminOrderResponse(
            Long id,
            String fileName,
            String studentName,
            String dateSubmitted,
            Long copies,
            String status
    ) {
        this.id = id;
        this.fileName = fileName;
        this.studentName = studentName;
        this.dateSubmitted = dateSubmitted;
        this.copies = copies;
        this.status = status;
    }

    public Long getId() {
        return id;
    }

    public String getFileName() {
        return fileName;
    }

    public String getStudentName() {
        return studentName;
    }

    public String getDateSubmitted() {
        return dateSubmitted;
    }

    public Long getCopies() {
        return copies;
    }

    public String getStatus() {
        return status;
    }
}

File: mobile/app/src/main/java/com/printit/mobile/features/admin/orders/AdminOrderResponse.kt
package com.printit.mobile.features.admin.orders

import com.google.gson.annotations.SerializedName

data class AdminOrderResponse(
    @SerializedName("id")
    val id: Long?,

    @SerializedName("fileName")
    val fileName: String?,

    @SerializedName("studentName")
    val studentName: String?,

    @SerializedName("dateSubmitted")
    val dateSubmitted: String?,

    @SerializedName("copies")
    val copies: Long?,

    @SerializedName("status")
    val status: String?
)